README.md

```diff
@@ -43,6 +43,10 @@ present you with the current PHP & WP-CLI versions installed if successful.
 mkdir your-wordpress-folder && cd your-wordpress-folder
 wp core download --locale=en_GB
 ```
+or if you have 'Oh my Zsh' installed you can simply use a shortcut:
```

Comment:
Can we add a line space above and below this text, just to keep it separate from the triple back ticks? It shouldn't cause any layout issues. 

Also perhaps we can rephrase this a little to explain what `take` does : 

Or if you have 'Oh My Zsh' installed, you can simplify this with the `take` command which will create and swap into the directory: